app/services/RestaurantService.java

```diff
@@ -30,228 +30,288 @@
-			restaurantReview = new RestaurantReview(
-					reviewForm.getRestaurantId(),
-					user.getId(),
-					reviewForm.getReviewScore(),
-					reviewForm.getReviewText()
-			);
-		} else {
-			restaurantReview.setReview(reviewForm.getReviewText());
-			restaurantReview.setRating(reviewForm.getReviewScore());
-		}
-
-		getSession().save(restaurantReview);
-		return true;
-	}
-
-	/**
-	 * Gets number of restaurants.
-	 *
-	 * @return the number of restaurants
-	 */
-	public Long getNumberOfRestaurants() {
-		return Long.valueOf(getSession().createCriteria(Restaurant.class)
-				.setProjection(Projections.rowCount())
-				.uniqueResult().toString());
-	}
-
-	/**
-	 * Update picture string.
-	 *
-	 * @param imageUploadForm the image upload form
-	 * @return the string
-	 * @throws Exception the exception
-	 */
-	public String updatePicture(final ImageUploadForm imageUploadForm) throws Exception {
-		Restaurant restaurant = (Restaurant) getSession().createCriteria(Restaurant.class)
-				.add(Restrictions.eq("id", imageUploadForm.getRestaurantId()))
-				.uniqueResult();
-
-		String newImagePath = AWS_BASE_PATH + imageUploadForm.getRestaurantId() + "-" + imageUploadForm.getImageType() + "." + imageUploadForm.getExtension();
-
-		if (imageUploadForm.getImageType().equals("profile")) {
-			restaurant.setProfileImagePath(newImagePath);
-		} else {
-			restaurant.setCoverImagePath(newImagePath);
-		}
-
-		getSession().update(restaurant);
-		return "{ \"imageFor\": \"" + imageUploadForm.getImageType() + "\", \"url\": \"" + newImagePath + "\"}";
-	}
+    private static final String AWS_BASE_PATH = "https://abhrestaurants.s3.amazonaws.com/";
+
+    @Inject
+    private RestaurantService() {
+    }
+
+    /**
+     * Create restaurant boolean.
+     *
+     * @param restaurant the restaurant
+     * @throws Exception the exception
+     */
+    public Boolean createRestaurant(final Restaurant restaurant) throws Exception {
+        getSession().save(restaurant);
+        return true;
+    }
+
+    /**
+     * Edit restaurant boolean.
+     *
+     * @param restaurant the restaurant
+     * @throws Exception the exception
+     */
+    public Boolean editRestaurant(final Restaurant restaurant) throws Exception {
+        getSession().merge(restaurant);
+        return true;
+    }
+
+    /**
+     * Delete restaurant boolean.
+     *
+     * @param id the id
+     * @throws Exception the exception
+     */
+    public Boolean deleteRestaurant(final UUID id) throws Exception {
+        Restaurant restaurant = (Restaurant) getSession().createCriteria(Restaurant.class)
+                .add(Restrictions.eq("id", id))
+                .uniqueResult();
+
+        getSession().delete(restaurant);
+        return true;
+    }
+
+    /**
+     * Find restaurants with filter pagination adapter.
+     *
+     * @param restaurantFilter the restaurant filter
+     * @return the pagination adapter
+     */
+    @SuppressWarnings("unchecked")
```

Comment:
I accidentally reformatted the code so now it looks that whole file is changed, but my actual changes are only in method findRestaurantsWithFilter().